backends/avx/ceed-avx-blocked.c

```diff
@@ -56,6 +57,8 @@ static int CeedInit_Avx(const char *resource, Ceed ceed) {
 // Backend Register
 //------------------------------------------------------------------------------
 CEED_INTERN int CeedRegister_Avx_Blocked(void) {
-  return CeedRegister("/cpu/self/avx/blocked", CeedInit_Avx, 30);
+  const char prefix[] = "/cpu/self/avx/blocked";
+  if (getenv("CEED_DEBUG")) fprintf(stderr, "Backend Register: %s\n", prefix);
+  return CeedRegister(prefix, CeedInit_Avx, 30);
```

Comment:
For a PR that claims to factorize code, it looks like the diffs in the 10 first files is just duplicated code, I think this calls for a function.

Reply:
I added `CeedDebugEnv()`